### PR TITLE
Add scripts for running GEOS-Chem v10-01 & v9-2 on YARCC

### PR DESCRIPTION
**Pull request by tsherwen**
I added the scripts that I am using to succesfully running the earlier version of GEOS-Chem on YARCC to my forked repository, but thought that others in the Evan's group would want to use these too.

### CODE REVIEW

File: scripts/setup_geos_environment_v10_v9_2.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+export GC_BIN="$GC_BASE/bin"
+export GC_INCLUDE="$GC_BASE/include"
+export GC_LIB="$GC_BASE/lib"
+
+################################################################################
+# OPENMP VARIABLES
+#-------------------------------------------------------------------------------
+# These must be set before running GEOS-Chem.
+################################################################################
+# Max out machine limits
```
Comment (kilicomu) [inline scripts/setup_geos_environment_v10_v9_2.sh]: Can you remove this comment? It is misleading in that `ulimit` does not 'max out machine limits', but controls resources available to a shell and all processes spawned by it.

File: scripts/setup_geos_environment_v11.sh [hunk 2]
```diff
@@ -50,4 +50,18 @@ export GC_F_LIB="$GC_F_BASE/lib"
 #-------------------------------------------------------------------------------
 # These must be set before running GEOS-Chem.
 ################################################################################
+# Max out machine limits
```
Comment (kilicomu) [inline scripts/setup_geos_environment_v11.sh]: Can you remove this comment? It is misleading in that `ulimit` does not 'max out machine limits', but controls resources available to a shell and all processes spawned by it.

File: scripts/setup_geos_environment_v11.sh [hunk 3]
```diff
@@ -50,4 +50,18 @@ export GC_F_LIB="$GC_F_BASE/lib"
...
+ulimit -d unlimited              # datasize
+ulimit -s unlimited              # stacksize
+ulimit -c unlimited              # coredumpsize
+ulimit -m unlimited              # memoryuse
+ulimit -v unlimited              # vmemoryuse
+ulimit -n unlimited              # descriptors
+ulimit -l unlimited              # memorylocked
+ulimit -u unlimited              # maxproc
+
+# Reset the child stack size to a large positive number
```
Comment (kilicomu) [inline scripts/setup_geos_environment_v11.sh]: Can you remove this comment and the comment on the line below? They don't accurately describe what `OMP_STACKSIZE` is used for.

**Comment by tsherwen** [comment]
I believe I have updated the PR as requested.

I also tested that GEOS-Chem ran without the ulimit settings.


Tomás Sherwen
<tomas.sherwen@gmail.com>
<https://uk.linkedin.com/pub/tomás-sherwen/26/857/a61> [image:
https://twitter.com/DrTomasS] <https://twitter.com/DrTomasS>
<https://scholar.google.co.uk/citations?user=e3SmNJoAAAAJ&hl>
<http://orcid.org/0000-0002-3006-3876>

**Pull request merged**